gef.py

```diff
@@ -7469,6 +7482,7 @@ class HexdumpCommand(GenericCommand):
 
     def __init__(self):
         super(HexdumpCommand, self).__init__(complete=gdb.COMPLETE_LOCATION)
+        self.repeat_count = 0
```

Comment:
This is fine for now, but I might consider adding this in the `GenericCommand`. Maybe set the `repeat_count` in `invoke`. That way all commands have `repeat_count` if they want it.